registry/handlers/app.go

```diff
@@ -212,10 +216,38 @@ func NewApp(ctx context.Context, configuration configuration.Configuration) *App
 // implementing this properly will require a refactor. This method may panic
 // if called twice in the same process.
 func (app *App) RegisterHealthChecks() {
-	health.RegisterPeriodicThresholdFunc("storagedriver_"+app.Config.Storage.Type(), 10*time.Second, 3, func() error {
+	health.RegisterPeriodicThresholdFunc("storagedriver_"+app.Config.Storage.Type(), defaultCheckInterval, 3, func() error {
```

Comment:
I avoided touching this to keep the scope of the PR limited, but it wouldn't be much work to add some options here. Does something like this look good?

    health:
      storagedriver:
        enabled: true
        interval: 10s
        threshold: 3

Without a "storagedriver" section, this would default to being enabled.